### PR TITLE
Fix typescript errors related to promises.

### PR DESCRIPTION
**Pull request by dborkan**
Fix typescript errors related to promises.

### CODE REVIEW

**Comment by keroserene** [comment]
LGTM

File: src/socks-to-rtc/socks.ts [hunk 1]
```diff
@@ -153,7 +153,7 @@ module Socks {
           })
           // Valid request - fire external callback.
           .then(this.maybeUdpStartRelay)
-          .then((request:any) => {
+          .then((request:any) : Promise<Channel.EndpointInfo> => {
```
Comment (iislucas) [inline src/socks-to-rtc/socks.ts:156]: Maybe I'm missing something, but why is the `:any` still there?

Comment (keroserene) [inline src/socks-to-rtc/socks.ts:156]: I don't think the `:any` is supposed to be there. But also, Trevor's pull request #59 changes a bunch of things, and I think this entire function disappears anyways.

Comment (iislucas) [inline src/socks-to-rtc/socks.ts:156]: cool.

On Mon, Mar 31, 2014 at 5:55 PM, Serene H notifications@github.com wrote:

> In src/socks-to-rtc/socks.ts:
> 
> > @@ -153,7 +153,7 @@ module Socks {
> >            })
> >            // Valid request - fire external callback.
> >            .then(this.maybeUdpStartRelay)
> > -          .then((request:any) => {
> > -          .then((request:any) : Promise<Channel.EndpointInfo> => {
> 
> I don't think the :any is supposed to be there. But also, Trevor's pull
> request #59 https://github.com/uProxy/socks-rtc/pull/59 changes a bunch
> of things, and I think this entire function disappears anyways.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/uProxy/socks-rtc/pull/68/files#r11139007
> .

## 

Lucas Dixon | Google Ideas

**Pull request merged**